bool create_collisions
int32[] ids
float32[] x
float32[] y
float32[] z
float32[] yaw
int32[] color
---
int32[] ids
float32[] x
float32[] y
float32[] z
float32[] yaw
int32[] color
